storage/object/how-to/get-started-edge-services.mdx

```diff
@@ -0,0 +1,293 @@
+
+The cache is now enabled.
+
+### How to purge all objects from your cache
+
+This clears all objects from your cache. Afterwards, fresh copies must be retrieved from the bucket before Edge Services can store them again in the cache. 
+
+1. Click **Object Storage** in the **Storage** section of the Scaleway console side menu. The list of your buckets displays.
+
+2. Click the name of the bucket you want to purge the cache for.
+
+3. Click the **Edge Services** tab.
+
+4. In the **Purge cache** panel, click **Purge all**. 
+
+        A screen displays warning you that your cache will be emptied, and Edge Services will have to retrieve objects from your bucket before re-caching them. 
+        
+5. Click **Purge cache** to confirm.
+
+### How to purge specific objects from your cache
+
+This allows you to specify the precise objects that you want to clear from the cache. Afterwards, fresh copies must be retrieved from the bucket before Edge Services can store them again in the cache. 
+
+1. Click **Object Storage** in the **Storage** section of the Scaleway console side menu. The list of your buckets displays.
+
+2. Click the name of the bucket you want to purge cache objects for.
+
+3. Click the **Edge Services** tab.
+
+4. In the **Purge cache** panel, click **Purge by object**. 
+        
+      A screen displays prompting you to enter the path of each object you want to purge from the cache.
+
+5. Enter the path of each object you want to purge. You can purge a maximum of 5 objects at a time.
+
+      <Message type="tip">
+
+      The path for each object should defined from the root of the bucket, starting with a slash. If we imagine a bucket containing one file at the root level called `object1.jpg`, and a subfolder at root level called `videos` containing an item called `my-video.mp4`, we would enter the object paths as follows:
+
+      - `/object1.jpg`
+      - `/videos/my-video.mp4`
+
+      You **cannot** purge entire subfolders by simply specifying the path to the subfolder, e.g. `/videos`. Purging objects must done strictly object-by-object, so `/videos/my-video1.mp4`, `/videos/my-video2.mp4`, `/videos/my-video3.mp4` etc. Watch this space for updates to this feature in the future.
+      </Message>
+
+6. Click **Purge objects from cache**
+
+    The specified objects are purged from your cache and you are returned to the Edge Services dashboard.
+
+### How to disable your cache
+
+1. Click **Object Storage** in the **Storage** section of the Scaleway console side menu. The list of your buckets displays.
+
+2. Click the name of the bucket you want to disable caching for.
+
+3. Click the **Edge Services** tab.
+
+    TODO SCREENSHOT
+
+4. In the **Cache** panel, use the <Icon name="toggle" /> icon to disable the cache.
+
+    A pop-up displays, asking you to confirm the action.
+
+5. Click **Disable cache**.
+
+    Your cache is disabled. Edge Services will now serve content by fetching it from your Object Storage bucket directly.
```

Comment:
@RoRoJ , maybe worth adding that the cache is also automatically purged, so if the user then re-enables cache he will start with an empty cache.